File: app/src/main/java/com/thewizrd/simpleweather/setup/SetupViewModel.java
package com.thewizrd.simpleweather.setup;

import androidx.lifecycle.ViewModel;

import com.thewizrd.shared_resources.locationdata.LocationData;

public class SetupViewModel extends ViewModel {
    private LocationData locationData;

    public SetupViewModel() {
        locationData = null;
    }

    public LocationData getLocationData() {
        return locationData;
    }

    public void setLocationData(LocationData locationData) {
        this.locationData = locationData;
    }
}

File: app/src/main/java/com/thewizrd/simpleweather/setup/SetupViewModel.kt
package com.thewizrd.simpleweather.setup

import androidx.lifecycle.SavedStateHandle
import androidx.lifecycle.ViewModel
import com.thewizrd.shared_resources.locationdata.LocationData

class SetupViewModel(private val state: SavedStateHandle) : ViewModel() {
    var locationData: LocationData? = null
}